### ISSUE TITLE
Not seeing logs remotely

### ISSUE THREAD

**Issue by anno73** [TheStaticTurtle/esphome_syslog#3]
Hi! 

I already remote log messages from my espeasy components. So basically the setup is working.

What I did was adding the settings to my esphome components yaml file:
```
debug:

logger:
  level: debug  # none, error, warn, info, debug, verbose, very_verbose

syslog:
    ip_address: "192.168.x.y"
    port: 514
```

I see, that it got compiled in the pio compile logs and after uploading mqtt logs give
```
[17:46:57][I][syslog:032]: Started
```
Still I do not see any logs in syslog. Anything I am missing out?

Also I do not understand in the description what your intentions with "then:" are. But this could also be because I am new to esphome.

Br, Alex.

**Comment by TheStaticTurtle** [comment]
Closing due to inactivity

**Comment by JDIacobbo** [comment]
For anyone who may come across this issue and have the same problem like I did, my cause was that my ESPHome devices were on a IoT vLAN and the messages out to the syslog server were being dropped due to firewall rules. I needed to move my device over to my main vLAN to get the logs into the syslog server. I assume I could have also added a rule to the firewall to allow communication but as this is just for testing I didn't bother.
